src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+    private static final int RESULT_INDEX = 2;
+    private static final int PARAM_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String[] params = {"supply", "buy", "result"};
```

Comment:
remove this variable, it's redundant

Reply:
Maybe I'll make a list of names in a separate Enum? I did this to reduce the number of constants.